ci/pipeline.yml

```diff
@@ -220,114 +226,28 @@ jobs:
             dir: src
             path: ci/tasks/restage.sh
         params:
-          <<: *staging-cf
-          CF_APP_NAME: pages-proxy-staging
-
-  - name: test-and-deploy-production
-    plan:
-      - get: src
-        resource: src-production
-        trigger: true
-        params: {depth: 1}
-      - put: gh-status
-        inputs: [src]
-        params: {state: pending}
-      - do: *test
-      - task: deploy
-        config:
-          <<: *cf-image
-          inputs: [name: src]
-          run:
-            dir: src
-            path: ci/tasks/deploy.sh
-        params:
-          <<: *production-cf
-          CF_APP_NAME: pages-proxy
-          CF_MANIFEST: .cloudgov/manifest.yml
-          CF_VARS_FILE: .cloudgov/vars/pages-production.yml
-        on_failure:
-          try:
-            task: cancel-api-deployment
-            config:
-              <<: *cf-image
-              inputs: [name: src]
-              run:
-                dir: src
-                path: ci/tasks/cancel-deployment.sh
-            params:
-              <<: *production-cf
-              CF_APP_NAME: pages-proxy
-    on_failure:
-      in_parallel:
-        - put: gh-status
-          inputs: [src]
-          params: {state: failure}
-        - put: slack
-          params:
-            text: |
-              :x: FAILED: pages proxy deployment on production
-              <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
-            channel: ((slack-channel))
-            username: ((slack-username))
-            icon_url: ((slack-icon-url))
-    on_success:
-      in_parallel:
-        - put: gh-status
-          inputs: [src]
-          params: {state: success}
-        - put: slack
-          params:
-            text: |
-              :white_check_mark: SUCCESS: Successfully deployed pages proxy on production
-              <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
-            channel: ((slack-channel))
-            username: ((slack-username))
-            icon_url: ((slack-icon-url))
-
-  - name: nightly-tasks-production
-    plan:
-      - get: src
-        resource: src-production
-        params: {depth: 1}
-      - get: nightly
-        trigger: true
-      - task: restage
-        config:
-          <<: *cf-image
-          inputs: [name: src]
-          run:
-            dir: src
-            path: ci/tasks/restage.sh
-        params:
-          <<: *production-cf
-          CF_APP_NAME: pages-proxy
+          <<: *env-cf
+          CF_APP_NAME: pages-proxy-((deploy-env))
 
 ############################
 #  RESOURCES
 
 resources:
 
-  - name: src-staging
-    type: git
-    icon: github
-    source:
-      uri: ((git-base-url))/((proxy-repository-path))
-      branch: staging
-
-  - name: src-production
+  - name: src-((deploy-env))
```

Comment:
It didn't complain about this? I swore this was giving me trouble before

Reply:
Maybe due to the `instance-var` feature being enabled recently.